docling_core/types/doc/document.py

```diff
@@ -1309,6 +1310,17 @@ def export_to_dict(self) -> Dict:
         """export_to_dict."""
         return self.model_dump(mode="json", by_alias=True, exclude_none=True)
 
+    def save_to_json_file(self, document_json_path, indent: int=4) -> int:
+        """export_to_json."""
+        with open(document_json_path, 'w') as f:
+            return f.write(self.model_dump_json(indent=indent))
+    
+    @classmethod
+    def load_from_json_file(cls, document_json_path):
```

Comment:
could you rename `document_json_path` to path and add the type `Path` and add the return type `-> DoclingDocument`.

I would also remove the assert.

Reply:
Done, thank you!